oxidation/libparsec/crates/core_fs/src/event_bus/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
```

Comment:
remove this mod and stick with a single `src/event_bus.rs`

Reply:
Done